bot/src/com/mygdx/game/client/network/MoveEntityService.java

```diff
@@ -30,10 +28,4 @@ public void moveEntity(int selectedUnit, Coordinates coordinates) {
     webSocket.send("move:" + selectedUnit + ":" + coordinates.getX() + ":" + coordinates.getY());
   }
 
-  @Data
```

Comment:
Jak tu wywalasz to mógłbyś także z desktopa

Reply:
wywalone to i CameraControl